### PR TITLE
[IGNORE FOR NOW - WIP] Mitigate Restart on RS5+

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

This is a mitigation on RS5+ for https://github.com/moby/moby/issues/38114#issuecomment-437956101. The kernel team here is tracking a resource leak issue for a (hopeful) future Windows update. For now, the only reliable way to get the RS5 CI servers running stable is to disable almost all of the restart tests.

### CODE REVIEW

**Comment by jhowardmsft** [comment]
Still some missing:

```
00:55:32.250 ERROR:  hcsshim::DestroyLayer - Win32 API call returned error r1=0x80070020 err=The process cannot access the file because it is being used by another process.id=CI flavour=0
00:55:32.305 
```

File: integration-cli/requirements_test.go [hunk 1]
```diff
@@ -69,6 +71,17 @@ func IsAmd64() bool {
 	return os.Getenv("DOCKER_ENGINE_GOARCH") == "amd64"
 }
 
+func NotWindowsRS5Plus() bool {
```
Comment (thaJeztah) [inline integration-cli/requirements_test.go]: Might be good to copy the comment you put  above `TestRestartStoppedContainer` here ("Certain tests here have testRequires(c, NotWindowsRS5Plus)....")

Comment (jhowardmsft) [reply integration-cli/requirements_test.go]: Sure, done. Think I have it now, had to disable another test. Got one clean run with a successful cleanup locally. Let's see how this goes on the production servers.

**Comment by olljanat** [comment]
> For now, the only reliable way to get the RS5 CI servers running stable is to disable almost all of the restart tests.

@jhowardmsft Side comment but is this issue only with case where same container stops and immediately restarts or would it happen also on case on swarm where new containers are created to replace existing ones?

**Comment by jhowardmsft** [comment]
@olljanat I'm not sure yet.

**Comment by thaJeztah** [comment]
Looks green; is this still WIP @jhowardmsft or ready for review?

**Pull request closed without merging**